packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -248,8 +245,7 @@ export default class PickerPlugin<T extends PickerDataProvider = PickerDataProvi
             } else {
                 this.setIsSuggesting(false);
             }
-        } else if (this.isCharacterValue) {
-            // Check for isCharacterValue to filter out modifiers like Ctrl+Z and Backspace

```

Comment:
You can still check if this is a text input event. Please reference to https://github.com/Microsoft/roosterjs/blob/63a7bf45faf61c1c1a2202d7302d1fd5610cda59/packages/roosterjs-editor-core/lib/coreAPI/attachDomEvent.ts#L16

If you want you can extract it into a util and put under https://github.com/Microsoft/roosterjs/tree/master/packages/roosterjs-editor-core/lib/eventApi

Reply:
Another option is to add a property to PluginKeyDownEvent and PluginKeyUpEvent since we have already checked it, no need to check again.

Reply:
Sounds good